src/Analyzers/CSharp/CodeFixes/UseCollectionInitializer/CSharpUseCollectionInitializerCodeFixProvider.cs

```diff
@@ -25,7 +23,7 @@ namespace Microsoft.CodeAnalysis.CSharp.UseCollectionInitializer
     using static SyntaxFactory;
 
     [ExportCodeFixProvider(LanguageNames.CSharp, Name = PredefinedCodeFixProviderNames.UseCollectionInitializer), Shared]
-    internal class CSharpUseCollectionInitializerCodeFixProvider :
+    internal partial class CSharpUseCollectionInitializerCodeFixProvider :
```

Comment:
this type changed a ton.  It is now 3 files.  The main one, with common logic.  And then two siblings.  One sibling is for creating object-initializers, the other for object-creation expressions.